x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/IndexPrivilege.java

```diff
@@ -82,7 +82,8 @@ public final class IndexPrivilege extends Privilege {
         // cross clusters query for ESQL
         "internal:data/read/esql/open_exchange",
         "internal:data/read/esql/exchange",
-        "indices:data/read/esql/cluster"
+        "indices:data/read/esql/cluster",
+        "cluster:monitor/xpack/enrich/esql/resolve_policy"
```

Comment:
this should be part of a named cross cluster *cluster* privilege (not the index privilege) : https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java#L166

Reply:
Yes, fixed in https://github.com/elastic/elasticsearch/pull/104879/commits/7cfeb7242e720871a87c571b02917096b3e53cc9.